src/node_config.cc

```diff
@@ -72,35 +72,38 @@ static void Initialize(Local<Object> target,
   READONLY_BOOLEAN_PROPERTY("hasTracing");
 #endif
 
-  READONLY_STRING_PROPERTY(target, "icuDataDir", icu_data_dir);
+  // TODO(addaleax): This seems to be an unused, private API. Remove it?
```

Comment:
I remember @srl295 saying this might be used by embedders.

Reply:
It’s only accessible through `process.binding('config')`, so it’s still going to have to change *somehow*, assuming we push through with the `process.binding()` deprecations…